### PR TITLE
Add context menu and several selection actions for the history list

### LINKED ISSUE #83: Selection actions in context menu
As a developer, I want to have several commands available to change the selection of the history list via a context menu that opens on right-click, because I want to facilitate different selection scenarios that would be tedious to accomplish otherwise.

**Acceptance criteria:**
* [ ] A context menu opens on a right-click on the history list (the one on the left).
* [ ] The context menu has the option "Ignore all", which deselects all changes in all methods.
* [ ] The context menu has the option "All latest changes", which selects the latest change of each method.
* [ ] The context menu has the option "Ignore all except this one", deselects all changes of all methods except the current method.

### PR DESCRIPTION
**Pull request by marcelgarus**
Fixes #83.
Merge #95 first.

### CODE REVIEW

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/ignoreAllOfHistoryListExceptSelected.st [hunk 1]
```diff
@@ -0,0 +1,9 @@
+menu-actions
+ignoreAllOfHistoryListExceptSelected
+
+	| selectedHistoryListItem |
+	selectedHistoryListItem := self selectedHistoryListItem.
```
Comment (gritfessel) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/ignoreAllOfHistoryListExceptSelected.st:5]: Why the temp var? In my opinion, `each = self selectedHistoryListItem` would be okay, too.

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/initializeHistoryList.st [hunk 2]
```diff
@@ -6,6 +6,4 @@ initializeHistoryList
 				showWhile: [(self retrieveChangeRecordsFromChangesFile
 						reject: [:each | self isInImage: each]) reversed].
 	self historyList: (self getHistoryListFrom: changeRecords).
-	self historyList do: [:each |
-		each history do: [:change |
-			change isSelected: true]]
\ No newline at end of file
+	self historyList do: [:each | each selectLatest]
```
Comment (gritfessel) [inline packages/ChangesReloaded-Core.package/CRBrowser.class/instance/initializeHistoryList.st:9]: Let's just use your cool new method for that :D
```suggestion
	self selectAllLatestChangesOfHistoryList
```

File: packages/ChangesReloaded-Core.package/CRHistoryListItem.class/instance/ignore.st [hunk 3]
```diff
@@ -0,0 +1,5 @@
+selecting
+ignore
+
+	self history do: [:change |
+		change isSelected: false]
```
Comment (gritfessel) [inline packages/ChangesReloaded-Core.package/CRHistoryListItem.class/instance/ignore.st:5]: I think the line wouldn't be too long.
```suggestion
	self history do: [:change | change isSelected: false]
```

Comment (strawberriesandcheese) [reply packages/ChangesReloaded-Core.package/CRHistoryListItem.class/instance/ignore.st:5]: do we have a rule for that one? I think we are doing it kinda too arbitrary right now :/

File: packages/ChangesReloaded-Core.package/CRBrowser.class/instance/ignoreAllOfHistoryListExceptSelected.st (hunk 1 above)
Comment (SilvanVerhoeven) [reply packages/ChangesReloaded-Core.package/CRBrowser.class/instance/ignoreAllOfHistoryListExceptSelected.st:5]: Historisch gewachsen

File: packages/ChangesReloaded-Core.package/CRHistoryListItem.class/instance/ignore.st (hunk 3 above)
Comment (SilvanVerhoeven) [reply packages/ChangesReloaded-Core.package/CRHistoryListItem.class/instance/ignore.st:5]: I'd be for the rule: New context, new line (it's code in a new block, so make that clear with a new line). Don't have too strong feelings about it tho

**Pull request merged**